include/date/solar_hijri.h

```diff
@@ -39,7 +42,7 @@ static const auto days_in_era = static_cast<unsigned>(1029983);
 static const auto years_in_era = static_cast<unsigned>(2820);
 static constexpr auto days_in_year = 365.24219858156028368;
 static constexpr auto leap_threshold = 0.24219858156028368;
-static const auto unix_time_shift = static_cast<unsigned>(2440587);
+static const auto unix_time_shift = static_cast<unsigned>(2440588);
```

Comment:
Are these constants part of the interface or just implementation details?  If the latter, how about stuffing them into a sub-namespace?

Reply:
Done!